lib/trino-parquet/src/main/java/io/trino/parquet/reader/ParquetReader.java

```diff
@@ -306,7 +306,9 @@ private boolean advanceToNextRowGroup()
             long rowCount = currentGroupRowRanges.getRowCount();
```

Comment:
commit comment
`... In this scenario, the reader is fixed to advance to next row group ...` -> `... In this scenario, the reader is fixed to advance to next matching row group ...`